server/cmd/main.go

```diff
@@ -58,6 +58,8 @@ func main() {
 	viper.AutomaticEnv()
 
 	// Meshery Server configuration
+	viper.SetConfigType("env")
```

Comment:
Why is it required? viper reads from the file specified in line 63.

Reply:
If configuration data can come from multiple sources also including environment variables, then these functions are required

Reply:
AutomaticEnv?

Reply:
> AutomaticEnv?

This is to automatically override and read env variables from the configuration file

Reply:
@MUzairS15  is this a desired change? Overkill, inconsequential? Should @anirudh-hegde be redirected to work on an existing issue or is this unacceptable change?